pennylane_ionq/ops.py

```diff
@@ -16,6 +16,68 @@
 """
 from pennylane.operation import Operation
 
+# Custom operations for the native gateset below.
+class GPI(Operation):  # pylint: disable=too-few-public-methods
+    r"""GPI(phi, wires)
+    Single-qubit GPI gate.
+    .. math::
+        GPI(\phi) =
+            \begin{pmatrix}
+                0 & e^{-i\phi} \\
+                e^{-i\phi} & 0
+            \end{pmatrix}
+    Args:
+        phi (float): phase :math:`\phi`
+        wires (Sequence[int]): the subsystems the operation acts on
+    """
+    num_params = 1
+    num_wires = 1
+    par_domain = "R"
```

Comment:
I believe `par_domain` is now a deprecated attribute, so we do not need to specify this anymore.
```suggestion
```

Reply:
Ah, cool!